tests/unit/utils/parsing_test.py

```diff
@@ -584,9 +627,10 @@ def test_parse_eyelink_blinks(tmp_path, metadata, expected_blinks):
             'END	10000024 	SAMPLES	EVENTS	RES	  38.54	  31.12\n',
             0.25,
             0.75,
-            id='missing_timestamps_lost_samples',
+            id='missing_timestamps_lost_samples4',
         ),
         pytest.param(
+            '** DATE: Wed Mar  8 09:25:20 2023\n'
```

Comment:
these added date strings aren't really necessary, right?
It really doesn't matter in this case and you don't need to revert them, but usually I would advise to avoid changes to existing test logic, e.g. changing test values.

Your other changes here, like adding documentation or changing test ids, are of course the spirit that we need! :1st_place_medal:

Reply:
Thanks for all your Feedback 😄 

The thing is, without this additional line, the metadata will be empty, since the MSG line gets parsed into the recording_config and not metadata anymore. However if metadata is empty/ none the code will raise a warning and all the tests, where I added the Date line,  will fail due to that. So I figured, I add a line that should be present in any dataset, which gets parsed by the metadata. 

Is there a better way, to solve this?